app/src/main/java/com/celzero/bravedns/service/PersistentState.kt

```diff
@@ -67,58 +69,47 @@ class PersistentState(private val context: Context) : SimpleKrate(context), Koin
     var tempBlocklistDownloadTime by longPref("temp_time_during_download", 0)
     var httpProxyPort by intPref("http_proxy_port", INVALID_PORT)
     var httpProxyHostAddress by stringPref("http_proxy_ipaddress", "")
-
     var killAppOnFirewall by booleanPref("kill_app_on_firewall", true)
     var allowByPass by booleanPref("allow_bypass", true)
     var allowDNSTraffic by booleanPref("dns_all_traffic", true)
     var dnsType by intPref("dns_type", 1)
     var prefAutoStartBootUp by booleanPref("auto_start_on_boot", true)
     var screenState by booleanPref("screen_state", false)
-
     var oldNumberRequests by intPref("number_request", 0)
     var oldBlockedRequests by intPref("blocked_request", 0)
     var numberOfRequests by longPref("dns_number_request", 0)
     var numberOfBlockedRequests by longPref("dns_blocked_request", 0)
-
     var backgroundEnabled by booleanPref("background_mode", false)
     var checkForAppUpdate by booleanPref("check_for_app_update", true)
-    private var connectedDnsName by stringPref("connected_dns_name", context.getString(R.string.dns_mode_3))
+    var connectedDnsName by stringPref("connected_dns_name", context.getString(R.string.dns_mode_3))
     var theme by intPref("app_theme", 0)
-    var notificationAction by intPref("notification_action", 1)
+    var notificationActionType by intPref("notification_action", 1)
```

Comment:
Are we storing user made notification action in persistent state? Why is that even required to be persisted? If for callbacks, then get rid. It should simply be a local state in `BraveVPNService`.